### PR TITLE
Add xidada.md

### PR DESCRIPTION
**Pull request by zwhmepsy**
Please review this, @casey-collab

### CODE REVIEW

File: Chinese/xidada/xidada.md [hunk 1]
```diff
@@ -0,0 +1 @@
+变成中国主席嘻嘻嘻
```
Comment (casey-collab) [inline Chinese/xidada/xidada.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by zwhmepsy** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**